lib/core/src/lib/card-view/components/card-view-dateitem/card-view-dateitem.component.ts

```diff
@@ -41,7 +40,8 @@ import { isValid } from 'date-fns';
     selector: 'adf-card-view-dateitem',
     templateUrl: './card-view-dateitem.component.html',
     styleUrls: ['./card-view-dateitem.component.scss'],
-    encapsulation: ViewEncapsulation.None
+    encapsulation: ViewEncapsulation.None,
+    host: { class: 'adf-card-view-dateitem' }
```

Comment:
should not be date-item instead of dateitem? We can also rename selector's name and files names

Reply:
closing basd on https://github.com/Alfresco/alfresco-ng2-components/pull/8995#discussion_r1379735582 comment as similar case